### DISCUSSION TITLE
Suspend option is missing in Session Indicator after upgrade

### DISCUSSION THREAD

**Discussion by dmocek** [elementary/wingpanel#477] [Q&A] [answered]
### What Happened?

On 3/10/2023, I ran
```
sudo apt update
sudo apt upgrade

```
I noticed that the indicator dropdown's look different.  Instead of text for shutdown, lock, and logout, there are now icons (which do look better).  However, suspend is now missing (or maybe I added it sometime back).  I use suspend daily.  Can it either be added back (preferred), or instructions provided as to how to add it?

### Steps to Reproduce

sudo apt update
sudo apt upgrade

### Expected Behavior

Suspend option to be available in the Session Indicator dropdown.

### OS Version

6.x (Odin/Jólnir)

### Software Version

Latest release (I have run all updates)

**Comment by danirabbit** [comment, marked as answer]
Hey thanks for your report! It sounds like you're in Early Access since that update hasn't rolled out to the stable release.

If you're using default settings, "Lock" does the same thing already so you can use "Lock" instead and your computer will suspend and lock.

If you want to suspend without locking, open System Settings → Security & Privacy → Locking and turn off "Lock on suspend". You'll then see a "Suspend" button in the session menu, but remember it will not lock your session when used.

**Comment by dmocek** [comment]
This worked for me.  Thanks!
